workspaces/liferay-partner-workspace/client-extensions/liferay-partner-etc-spring-boot/src/main/java/com/liferay/partner/QueueListener.java

```diff
@@ -787,12 +845,24 @@ private void _updateAccountRegion(
 	private static final String _ACCOUNT_ROLE_PARTNER_SALES_USER_NAME =
 		"[Account] Partner Sales User (PSU)";
 
+	private static final String _ACCOUNT_ROLE_TECHNICAL_SALES_USER_NAME =
+		"[Account] Partner Technical User (PTU)";
+
+	private static final String _KORONEIKI_ROLE_MARKETING_USER_NAME =
+		"Partner Marketing User";
+
 	private static final String _KORONEIKI_ROLE_PARTNER_MANAGER_NAME =
 		"Partner Manager";
 
 	private static final String _KORONEIKI_ROLE_PARTNER_MEMBER_NAME =
 		"Partner Member";
 
+	private static final String _KORONEIKI_ROLE_PARTNER_SALES_USER_NAME =
+		"Partner Sales User";
+
+	private static final String _KORONEIKI_ROLE_TECHNICAL_USER_NAME =
```

Comment:
Can you make this var name PARTNER_TECHNICAL_USER to match the actual name better